### PR TITLE
HID: add HORI mini wired PS4 USB controller

### PR DESCRIPTION
**Pull request by Kert**
Related to https://github.com/libretro/RetroArch/pull/11280

### CODE REVIEW

**Review by RobLoach — CHANGES_REQUESTED** [review]
Thanks a lot! Would you mind adding a comment at the top of the config with a link to the hardware?

**Comment by Kert** [comment]
@RobLoach Would this be fine? https://stores.horiusa.com/wired-mini-gamepad-black/. Does // work there?
Also the other configs don't have any links included

**Comment by RobLoach** [comment]
That URL is perfect. You'll want a comment about the device, along with a `input_device_display_name` so that the notification looks pretty when you plug it in.......

Example: https://github.com/libretro/retroarch-joypad-autoconfig/blob/master/udev/3H%20Dual%20Arcade.cfg

File: hid/HORI_mini_wired_ps4.cfg [hunk 1]
```diff
@@ -0,0 +1,55 @@
+input_device = "HORI mini wired PS4"
```
Comment (RobLoach) [inline hid/HORI_mini_wired_ps4.cfg:1]:
``` suggestion
# HORI Wired Mini Gamepad for PS4
# https://stores.horiusa.com/wired-mini-gamepad-black/
input_device = "HORI mini wired PS4"
input_device_display_name = "HORI PS4 Mini Wired Gamepad"
```

**Pull request merged**